src/component/button/button.scss

```diff
@@ -56,6 +56,26 @@
       }
     }
 
+    &.catalog {
+      border: 1px solid themed('secondary-color');
+      color: themed('primary-text');
+      border-radius: 12px;
+      background: themed('secondary-color');
+      box-shadow: 0 4px 12px rgba(255, 149, 0, 0.6);
+
+      transition: background-color 200ms ease-in-out;
```

Comment:
as this transition is used twice in the app, I think you can move it into the `mixins` file to avoid code repetition